### PR TITLE
Updated README.md to include GNUPG2 installation for setup on Windows WSL

### LINKED ISSUE #11464: Error while adding GPG key for project installation in Windows subsystem for linux
### Error while adding GPG key for project installation in Windows subsystem for linux

In the documentation on installing the project on Windows subsystem for Linux, while trying to add the GPG keys, running `gpg2 --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB` . 
This returns an errror `Command gpg2 not found`

Here is a snapshot of the error
image

### Look in
https://github.com/publiclab/plots2/blob/main/README.md

### Browser, version, and operating system
Windows subsystem for linux - Ubuntu 20.04

### PR DESCRIPTION
**Pull request by RealRichi3**
Added `sudo apt install gnupg2` to the project installation guide (README.md) for Windows subsystem for linux

Fixes #11464 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by RealRichi3** [comment]
@jywarren please review my request

File: README.md [hunk 1]
```diff
@@ -135,6 +135,7 @@ Before continuing with the installation steps in this README, users of Windows S
   * `sudo apt install libyaml-dev make pkg-config sqlite3 zlib1g-dev libgmp-dev`
   * `sudo apt install libreadline-dev libssl-dev`
 2. Add GPG Key & Install RVM:
+  * `sudo apt install gnupg2`
```
Comment (cesswairimu) [inline README.md:138]: Thanks for opening this @RealRichi3, its looking good. Maybe we should add some text before the command smth like "Install gnupg2 if you haven't already: "  What do you think?

Comment (RealRichi3) [reply README.md:138]: True, I'll add that

Comment (RealRichi3) [reply README.md:138]: Or better still, since I've already contributed, let me make it a `first-time-issue` so others can fix it

Comment (cesswairimu) [reply README.md:138]: Sounds great  :+1: . Thanks

**Review by cesswairimu — APPROVED** [review]

**Pull request merged**